src/MCPClient/lib/clientScripts/create_transfer_mets.py

```diff
@@ -197,6 +208,33 @@ def lookup_file_data(relative_path, db_base_path, lookup_kwargs):
     return file_obj
 
 
+def lookup_file_rights(file_uuid, transfer_uuid):
+    TRANSFER_RIGHTS_LOOKUP_UUID = "45696327-44c5-4e78-849b-e027a189bf4d"
+    FILE_RIGHTS_LOOKUP_UUID = "7f04d9d4-92c2-44a5-93dc-b7bfdf0c1f17"
```

Comment:
I'd prefer not to use hardcoded UUIDs here, but that's how the model is. In the meantime, I'd prefer to move this up to module level, but` ensure_no_mutable_globals` complains.